R/apa_print_anova.R

```diff
@@ -4,17 +4,29 @@
 #' strings to report the results in accordance with APA manuscript guidelines. For \code{anova}-objects from model comparisons see \code{\link{apa_print.list}}.
 #'
 #' @param x Output object. See details.
-#' @param correction Character. In the case of repeated-measures ANOVA, the type of sphericity correction to be used (\code{GG} for Greenhouse-Geisser or \code{HF} for Huyn-Feldt methods or \code{none}). Default is \code{GG}.
-#' @param intercept Logical. Indicates if intercept test should be included in output.
-#' @inheritParams print_anova
-#' @inheritDotParams print_anova
+#' @param correction Character. In the case of repeated-measures ANOVA, the type of sphericity correction to be used (\code{"GG"} for Greenhouse-Geisser or \code{"HF"} for Huyn-Feldt methods or \code{"none"}). Default is \code{"GG"}.
+#' @param intercept Logical. Indicates if the intercept term should be included in output.
+#' @param estimate Character, function, or data frame. Determines which estimate of effect size is to be used. See details.
+#' @param mse Logical. Indicates if mean squared errors should be included in output. Default is \code{TRUE}.
+#' @param observed Character. The names of the factors that are observed, (i.e., not manipulated). Necessary for calculation of generalized eta-squared; otherwise ignored.
+#' @param in_paren Logical. Indicates if the formatted string will be reported inside parentheses. See details.
 #' @details
 #'    The factor names are sanitized to facilitate their use as list names (see Value section). Parentheses
 #'    are omitted and other non-word characters are replaced by \code{_}.
 #'
 #'    If \code{in_paren} is \code{TRUE} parentheses in the formatted string, such as those surrounding degrees
 #'    of freedom, are replaced with brackets.
 #'
+#'    Argument `estimate` determines which measure of effect size is to be used: It is currently possible to provide
+#'    a character, where `"ges"` calculates generalized eta squared,`"pes"` calculates partial eta squared, and `"es"`
+#'    calculates eta squared. Note that eta squared is calculated correctly if and only if the design is balanced.
+#'
+#'    It is also possible to provide a `data frame` with columns `estimate`, `conf.low`, and `conf.high`, which allows
+#'    for including custom effect-size measures.
+#'
+#'    A third option is to provide a function that will be used to calculate effect-size measures from `x`. See the
```

Comment:
What do you mean in addition to "with columns `estimate`, `conf.low`, and `conf.high`"?

Reply:
I'm referring to the third option here.

Reply:
We only support **effectsize** right now, so I would keep it like that. I would not encourage users to write their own functions that we call from within `apa_print()`, because then we would have to maintain tidiers for these functions. If a user wants to write their own function to calculate effect sizes, their function should most likely just create an output that is understood by the data-frame method.

Reply:
Okay, so should we change this to "a function from the effectsize package" to make it crystal clear?

Reply:
done